models/base.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+import "time"
+
+type Base struct {
+	Id        int64     `orm:"auto"`
```

Comment:
> `orm:"auto"`
I think we dont need it as Beego orm auto increment the id  by default

Reply:
Removed ✌️